src/Migration/Sources/Appwrite/Reader/Database.php

```diff
@@ -24,94 +24,96 @@ public function __construct(private readonly UtopiaDatabase $dbForProject)
     {
     }
 
-    public function report(array $resources, array &$report): void
+    public function report(array $resources, array &$report): mixed
```

Comment:
Why return mixed? Can still be void I think

Reply:
pretty sure the IDE complained for void while the underlying abstract method return is mixed 🤔